config.go

```diff
@@ -65,6 +65,6 @@ type State struct {
 	ID string `json:"id"`
 	// Pid is the process id for the container's main process.
 	Pid int `json:"pid"`
-	// Root is the path to the container's bundle directory.
-	Root string `json:"root"`
+	// Bundle is the path to the container's bundle directory.
+	Bundle string `json:"bundle"`
```

Comment:
nit: How about `BundlePath` since `Bundle` can be interpreted as the name of the bundle.